### ISSUE TITLE
Troubles making it work

### ISSUE THREAD

**Issue by aabafra** [bieniu/ha-shellies-discovery-gen2#133]
Hi!

I've always manually integrated my Shelly devices into HASS thru MQTT but I'm finding issues with gen2 devices.

I'm trying this script with a device configured like this:

![image](https://user-images.githubusercontent.com/63967859/209820188-41b207fb-12ed-451c-a6bc-0a1e265b33ea.png)

I've added the following 2 automations:

```
- id: shellies_announce_gen2
  alias: "Shellies Announce Gen2"
  trigger:
    - platform: homeassistant
      event: start
  variables:
    device_info_payload:  "{{ {'id': 1, 'src':'shellies_discovery', 'method':'Shelly.GetConfig'} | to_json }}"
    device_ids:  # enter the list of device IDs here
      - "shellies/luz-pasillo-techo"
  action:
    - repeat:
        for_each: "{{ device_ids }}"
        sequence:
          - service: mqtt.publish
            data:
              topic: "{{ repeat.item }}/rpc"
              payload: "{{ device_info_payload }}"

- id: shellies_discovery_gen2
  alias: "Shellies Discovery Gen2"
  mode: queued
  max: 999
  trigger:
  - platform: mqtt
    topic: shellies_discovery/rpc
  action:
  - service: python_script.shellies_discovery_gen2
    data:
      id: "{{ trigger.payload_json.src }}"
      device_config: "{{ trigger.payload_json.result }}"
```
I have placed the script at python_scripts folder. 

But I don't get it working. Nothing happens at all. 

Any idea where the problem could be?

**Comment by aabafra** [comment]
I got these logs:

> Logger: homeassistant.components.python_script
> Source: components/python_script/__init__.py:112
> Integration: Python Scripts ([documentation](https://www.home-assistant.io/integrations/python_script), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+python_script%22))
> First occurred: 5:29:58 PM (25 occurrences)
> Last logged: 5:31:04 PM
> 
> Error loading script shellies_discovery.py: Line 71: SyntaxError: invalid character '·' (U+00B7) at statement: '<title>ha-shellies-discovery/shellies_discovery.py at master · bieniu/ha-shellies-discovery</title>'
> Error loading script shellies_discovery_gen2.py: Line 71: SyntaxError: invalid character '·' (U+00B7) at statement: '<title>ha-shellies-discovery-gen2/shellies_discovery_gen2.py at master · bieniu/ha-shellies-discovery-gen2 · GitHub</title>'

**Comment by bieniu** [comment]
Why didn’t you fill up the issue form?
The content of py files is html code. Your installation is completely wrong.

**Comment by aabafra** [comment]
Sorry for that. I had already written the issue and that's why I used a blank template.

I downloaded the py file making "righlt click" here: https://github.com/bieniu/ha-shellies-discovery-gen2/tree/master/python_scripts

And I don't know why it was downloaded as an HTML kind file (with py extension).

I've installed both scripts directly through HACS and they now work perfectly.

These scripts are life-changing. Thanks for them!
